src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php

```diff
@@ -24,16 +24,20 @@
 
 /**
  * @author SpacePossum
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
 final class PhpUnitDedicateAssertFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
     private static $fixMap = [
         'array_key_exists' => ['assertArrayNotHasKey', 'assertArrayHasKey'],
         'empty' => ['assertNotEmpty', 'assertEmpty'],
```

Comment:
this fixer handle only `assertTrue(method(...))` and `assertFalse(method(...))`

Reply:
I left `assertCount` out by design because of an issue with HHVM and PHPUnit IIRC.
Since we dropped HHVM support officially, we could add it to the fixer (not in this PR but somewhere in the future)